src/helpers/user/user-helper.js

```diff
@@ -8,6 +9,145 @@ const principalStatusApiMap = {
   Inactive: 'disabled',
   All: 'all',
 };
+
+const getBaseUrl = (url) => {
+  if (isInt) {
+    return url.int;
+  } else if (isStage) {
+    return url.stage;
+  } else if (isITLessProd) {
+    return url.prod;
+  } else {
+    return '';
+  }
+};
+
+async function fetchBaseUrl() {
+  try {
+    const response = await fetch(`${insights.chrome.isBeta() ? '/beta' : ''}/apps/rbac/env.json`);
+    const jsonData = await response.json();
+    return jsonData;
+  } catch (error) {
+    console.log(error);
+  }
+}
+
+export async function addUsers(usersData = { emails: [], isAdmin: undefined }) {
+  const token = await insights.chrome.auth.getToken();
+  const requestOpts = {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+      Accept: 'application/json',
+      Authorization: `Bearer ${token}`,
+    },
+    body: JSON.stringify({
+      emails: usersData.emails,
+      isAdmin: usersData.isAdmin,
+    }),
+  };
+  const url = await fetchBaseUrl();
+  const baseUrl = getBaseUrl(url);
+  let promise = new Promise((resolve, reject) => {
+    return fetch(`${baseUrl}/user/invite`, requestOpts)
```

Comment:
Is it possible to use the generated JS clients? We can sue the v2 which now has a proper tree shaking. Or is this endpoint specific to fedramp?

Reply:
This is keycloak api urls specific to Fedramp and not to be exposed on commercial